gobot_msgs/BatteryMsg battery

gobot_msgs/BumperMsg bumper

std_msgs/Int8 button

bool charging

std_msgs/Int8 reset_button

gobot_msgs/CliffMsg cliff

std_msgs/Int8 magnet

gobot_msgs/GyroMsg gyro

gobot_msgs/IrMsg infrared

gobot_msgs/ProximityMsg proximity

gobot_msgs/SonarMsg sonar

std_msgs/Float32 temperature

gobot_msgs/WeightMsg weight